terra-jupyter-aou/Dockerfile

```diff
@@ -133,8 +133,8 @@ RUN R -e 'BiocManager::install(c( \
   "SAIGEgds", \
   "GENESIS"))'
 
-RUN pip3 install --upgrade \
-  nbstripout \
-  "git+git://github.com/all-of-us/workbench-snippets.git#egg=terra_widgets&subdirectory=py" \
+RUN pip3 install --upgrade nbstripout \
   && mkdir -p /home/$USER/.config/git \
-  && nbstripout --install --global
+  && nbstripout --install --global \
+  # The base image already contains the needed dependencies for this package.
```

Comment:
This feels like it may be introducing a brittle dependency, you can imagine in the future workbench-snippets might import a package which is not installed in the base image - it seems correct to allow workbench-snippets to define its own package dependencies

Can we instead just remove `--upgrade` from the install line? Would that also resolve the issue?

Reply:
updated!

If later we have some packages that we know we want upgraded, we can add another pip command.